pkg/ec2/ec2.go

```diff
@@ -69,6 +70,9 @@ func CreateInstance(client *ec2.EC2, ec2Params Params) (*ec2.Reservation, error)
 				},
 			},
 		},
```

Comment:
Let's capitalize all instances of Iam. For example: `IamInstanceProfile` -> `IAMInstanceProfile`

Reply:
Whoopsie, good catch! Looks like they didn't adhere to best practices, but it's not our problem :)